lib/modules/platform/gerrit/client.ts

```diff
@@ -213,6 +214,13 @@ class GerritClient {
     );
   }
 
+  normalizeMessage(message: string): string {
+    const trimmedMessage = message.trim(); //the last \n was removed from gerrit after the comment was added...
+    return trimmedMessage.length > 0x4000
+      ? trimmedMessage.slice(0, 0x4000)
+      : trimmedMessage;
+  }
```

Comment:
remove condition and use substring

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring